pkg/shell/index.html

```diff
@@ -358,6 +358,10 @@ <h4 class="modal-title" translatable="yes">Authentication</h4>
                         </ul>
                     </div>
                 </div>
+                <div class="has-feedback search" id="system-search">
```

Comment:
Can we call this "menu-search" or even better, "manifest-search", to not confuse it with the System page?